src/main/java/org/kuali/kra/irb/ProtocolAction.java

```diff
@@ -96,14 +99,15 @@
     private static final String NOT_FOUND_SELECTION = "The attachment was not found for selection ";
     private static final ActionForward RESPONSE_ALREADY_HANDLED = null;
     private static final String INVALID_WATERMARK = "Watermark the attachment to invalid due to attachment status or doc status.";
-    
-    
+
+
     private WatermarkService watermarkService = null;
     private ProtocolPrintingService protocolPrintingService = null;
-       
+    private ProtocolAttachmentService protocolAttachmentService = null;
```

Comment:
Declaring an instance variable to null is redundant -- was this just to follow the style of the other's in this class?